src/main/java/javax/jmdns/impl/DNSRecord.java

```diff
@@ -571,8 +571,8 @@ public ServiceEvent getServiceEvent(JmDNSImpl dns) {
         protected void toString(final StringBuilder sb) {
             super.toString(sb);
             sb.append(" alias: '")
-                .append(_alias != null ? _alias : "null")
-                .append('\'');
+                    .append(_alias)
```

Comment:
Why is this changing the indentation?

Reply:
It wasn't intentional, but my IDE puts it there.

Reply:
Now fixed.